local/postman/TX-Training.postman_collection.json

```diff
@@ -0,0 +1,1851 @@
+								"exec": [
+									"var plain = pm.collectionVariables.get(\"PLAIN_QUERY\")",
+									"var encoded = Buffer.from(plain).toString(\"base64\")",
+									"pm.collectionVariables.set(\"ENCODED_QUERY\", encoded)"
+								],
+								"type": "text/javascript",
+								"packages": {}
+							}
+						}
+					],
+					"request": {
+						"auth": {
+							"type": "bearer",
+							"bearer": [
+								{
+									"key": "token",
+									"value": "{{BEARER_TOKEN_SUPPLIER}}",
+									"type": "string"
+								}
+							]
+						},
+						"method": "GET",
+						"header": [
+							{
+								"key": "Edc-Bpn",
+								"value": "BPNL4444444444XX",
+								"description": "BPN (This is normally send via EDC)",
+								"type": "default"
+							}
+						],
+						"url": {
+							"raw": "http://dtr-supplier:4243/api/v3/shell-descriptors/{{ENCODED_SHELL_ID}}",
+							"protocol": "http",
+							"host": [
+								"dtr-supplier"
+							],
+							"port": "4243",
+							"path": [
+								"api",
+								"v3",
+								"shell-descriptors",
+								"{{ENCODED_SHELL_ID}}"
+							]
+						}
+					},
+					"response": []
+				}
+			]
+		},
+		{
+			"name": "Step 2",
+			"item": [
+				{
+					"name": "Obtain Token for SupplierDTR",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"var token = pm.response.json()[\"access_token\"].toString().trim()",
+									"pm.collectionVariables.set(\"BEARER_TOKEN_SUPPLIER\", token)"
+								],
+								"type": "text/javascript",
+								"packages": {}
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "grant_type",
+								"value": "client_credentials",
+								"type": "text",
+								"disabled": true
+							},
+							{
+								"key": "client_id",
+								"value": "FOSS-DTR-CLIENT",
+								"type": "text",
+								"disabled": true
+							},
+							{
+								"key": "client_secret",
+								"value": "oPvRzIFZ1g1qgVXLzM7goQKrmh7IAilG",
+								"type": "text",
+								"disabled": true
+							}
+						],
+						"body": {
+							"mode": "urlencoded",
+							"urlencoded": [
+								{
+									"key": "grant_type",
+									"value": "client_credentials",
+									"type": "text"
+								},
+								{
+									"key": "client_id",
+									"value": "FOSS-DTR-CLIENT",
+									"type": "text"
+								},
+								{
+									"key": "client_secret",
+									"value": "TESTUSERPASSWORD_123",
+									"type": "text"
+								}
+							]
+						},
+						"url": {
+							"raw": "http://keycloak:8080/realms/Supplier/protocol/openid-connect/token",
+							"protocol": "http",
+							"host": [
+								"keycloak"
+							],
+							"port": "8080",
+							"path": [
+								"realms",
+								"Supplier",
+								"protocol",
+								"openid-connect",
+								"token"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "PostNewShell to SupplierDTR",
```

Comment:
No the supplier does not know it. 

But the Non-Scenario Customer does --> BPNL2222222222RR

And since the customers DTR knows that BPNL, we could use that BPNL within a postman request, if we want to. 
We would need to hardcode the password for retrieving a token for the customer into the .env file, but that no big deal

Reply:
Sounds great!